inference-engine/ie_bridges/python/src/openvino/inference_engine/__init__.py

```diff
@@ -19,17 +19,28 @@
 import sys
 
 if sys.platform == "win32":
-    # PIP installs openvino dlls 3 directories above in openvino.libs by default
+    # Installer, yum, pip installs openvino dlls to the different directory
     # and this path needs to be visible to the openvino modules
     #
     # If you're using a custom installation of openvino,
     # add the location of openvino dlls to your system PATH.
-    openvino_dlls = os.path.join(os.path.dirname(__file__), "..", "..", "openvino", "libs")
-    if (3, 8) <= sys.version_info:
-        # On Windows, with Python >= 3.8, DLLs are no longer imported from the PATH.
-        os.add_dll_directory(os.path.abspath(openvino_dlls))
-    else:
-        os.environ["PATH"] = os.path.abspath(openvino_dlls) + ";" + os.environ["PATH"]
+    openvino_libs = [
+            '../../../../deployment_tools/inference_engine/bin/intel64/Release',
```

Comment:
I am not sure that it is a good idea to use relative paths without any flexibility here.
I think it will be better to use some advance logic instead of hardcoded paths.

Reply:
We may check %INTEL_OPENVINO_DIR% (case when used setupvars.bat) but we would still need to hardcode installation subdirs...

Reply:
Used variables from setupvars script instead